BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -237,10 +237,12 @@ - (void)didReceiveAccessTokenRefreshedNotification:(NSNotification *)notificatio
 
 - (void)didReceiveUserWasLoggedOutNotification:(NSNotification *)notification
 {
-    NSDictionary *userInfo = (NSDictionary *)notification.object;
-    NSString *userID = [userInfo objectForKey:BOXUserIDKey];
-    if ([userID isEqualToString:self.user.modelID]) {
-        [self logOut];
+    if ([notification.object isKindOfClass:[NSDictionary class]]) {
```

Comment:
Agreed 100%. We had that discussion, that is IOS-12164. Thuy did not want to do that for 3.8.X releases, so we'll do it in Master for Hamilton. I want to do this next week.